beta-component-promotion/promotion-2022.14/beta-promotion-candidates.md

```diff
@@ -7,7 +7,7 @@ If you don't want multiple alerts to stack and fill the screen until the user di
 ![alert group overflow](./img/alert-group-overflow.png)
 
 ### [Calendar month](https://www.patternfly.org/v4/components/calendar-month)
-A calendar month component allows users to select and navigate between days, months and/or years. This component is usually used with the date picker component, to display date options and selections.
+A calendar month com. This component is usually used with the date picker component, to display date options and selections.
```

Comment:
I think this is a typo

Reply:
Fixed later here https://github.com/patternfly/patternfly-org/pull/3557